lms/services/user.py

```diff
@@ -1,7 +1,7 @@
 from functools import lru_cache
-from typing import Optional
 
 from sqlalchemy.exc import NoResultFound
+from sqlalchemy.orm import Query
```

Comment:
`Query` is deprecated or at least "legacy" (although apparently it's not actually being removed): <https://docs.sqlalchemy.org/en/20/orm/queryguide/query.html#sqlalchemy.orm.Query>

Since SQLAlchemy 1.4 I think you want `select()` instead.